src/main/java/com/amazon/opendistroforelasticsearch/security/OpenDistroSecurityPlugin.java

```diff
@@ -189,6 +194,18 @@
     private volatile DlsFlsRequestValve dlsFlsValve = null;
     private volatile Salt salt;
 
+    public static boolean isActionTraceEnabled() {
```

Comment:
Why not move to a separate class to fully encapsulate the logic ?

Reply:
The functionality is too small to justify for a separate class, IMO.

Reply:
I agree these 3 functions may be small but I dont think OpenDistroSecurityPlugin is the right place for this.

Reply:
Please suggest which class will be a better place to move those methods (new class is not an option).

Reply:
Curious as to why new class is not an option?

Reply:
- The number of new lines of code that will be necessary to introduce with a new class (class declaration and private constructor) would be comparable to number of lines introduced in these 3 methods.
- It helps to avoid using more memory in JVM metaspace (not a big deal)

Reply:
IMO different functionality should warrant a separate class even if it is small and should not be comparing the number of lines changed. 
Isn't the goal of this PR to encapsulate the logic for action tracer ? 

Just curious to know how much more memory we are talking about

Reply:
The functionality is embedded into the `Logger actionTrace` member variable, there is no need to have a separate class for it.

The primary goal of the PR is to avoid excessive calls to `LogManager.getLogger()` in various places and especially every time `OpenDistroSecurityRequestHandler` instance is created.

> Just curious to know how much more memory we are talking about

Not much, but we can leave this memory to other classes where a separate class is better justified.